website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -26,7 +26,11 @@ To add an application to authentik and have it display on users' **My applicatio
 
 ## Authorization
 
-Application access can be configured using (Policy) bindings. Click on an application in the applications list, and select the _Policy / Group / User Bindings_ tab. There you can bind users/groups/policies to grant them access. When nothing is bound, everyone has access. You can use this to grant access to one or multiple users/groups, or dynamically give access using policies.
+Application access can be configured using either (Policy) bindings or Application Entitlements.
+
+### Policy-driven authorization
+
+To use a policy to control which users or groups can access an application, click on an application in the applications list, and select the **Policy/Group/User Bindings** tab. There you can bind users/groups/policies to grant them access. When nothing is bound, everyone has access. You can use this to grant access to one or multiple users/groups, or dynamically give access using policies.
```

Comment:
Question: "When nothing is bound, everyone has access. You can use this to grant access..."

That confuses me. Aside from the antecedent issue (what does "this" refer to in that sentence?), if everyone has access, why would I want to grant _more_ access?  

> When nothing is bound, everyone has access.  Binding a policy restricts access to specifiec Users or Groups, or by other custom policies such as restriction to a set time-of-day or a geographic region.

Reply:
whoa, very cool examples of ways ot use policies to restrict access, thank you!